src/views/settings/EditPlayerDsp.vue

```diff
@@ -10,6 +10,49 @@
       <v-toolbar-title>{{
         $t("settings.dsp.configure_on", { name: playerName })
       }}</v-toolbar-title>
+      <v-spacer />
+      <v-menu offset-y transition="slide-y-transition">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            v-bind="props"
+            class="mr-4"
+            :class="getButtonClass()"
+            prepend-icon="mdi-tray-arrow-down"
+          >
+            {{ $t("settings.dsp.presets.load") }}
+          </v-btn>
+        </template>
+        <v-list>
+          <v-list-item v-if="dspPresets.length === 0">
+            <v-list-item-title>
+              {{ $t("settings.dsp.presets.empty_warning") }}
+            </v-list-item-title>
+          </v-list-item>
+          <v-list-item
+            v-else
+            v-for="preset in dspPresets"
+            :key="preset.preset_id"
+            @click="loadPreset(preset)"
+          >
+            <v-list-item-title>{{ preset.name }}</v-list-item-title>
+            <template #append>
+              <v-btn
+                icon="mdi-delete"
+                variant="text"
+                density="compact"
+                @click.stop="removePreset(preset.preset_id)"
+              />
+            </template>
+          </v-list-item>
+        </v-list>
+      </v-menu>
+      <v-btn
+        :class="getButtonClass()"
+        prepend-icon="mdi-content-save"
+        @click="showSavePresetDialog = true"
+      >
+        {{ $t("settings.dsp.presets.save") }}
```

Comment:
```suggestion
        <span class="d-none d-md-inline">
          {{ $t("settings.dsp.presets.save") }}
        </span>
```
Show just the icon for smaller screens like phones.